shell/common/rasterizer.cc

```diff
@@ -567,7 +567,10 @@ RasterStatus Rasterizer::DrawToSurfaceUnsafe(
     frame_timings_recorder.RecordRasterStart(fml::TimePoint::Now());
 
     std::unique_ptr<FrameDamage> damage;
-    if (frame->framebuffer_info().supports_partial_repaint) {
+    // when leaf layer tracing is enabled we wish to repaing the whole frame for
```

Comment:
typo?

I also think this is probably going to make the performance metrics less helpful. For example, imagine you have a relatively expensive leaf (some kind of fancy drawing or SVG or something) that does not change from frame to frame. Enabling leaf layer tracing will make it artificially seem like that leaf layer is very expensive on every frame, when it's really only very expensive on the first one (maybe where you still have enough budget for it).